src/joystick_interrupt.cpp

```diff
@@ -75,6 +77,15 @@ class JoystickInterrupt
         }
       }
 
+      if (high_speed_button_ >= 0)
+      {
+        if (msg->buttons[high_speed_button_])
+        {
+          lin *= high_speed_ratio_;
+          ang *= high_speed_ratio_;
```

Comment:
Multiplying it to both linear and angular velocity might be unsafe.
Since centrifugal acceleration, which can cause rollover, is a product of linear vel and angular vel, `high_speed_ratio_` effects by square.
This would be better to be two different parameters.

Reply:
Done